aiida/cmdline/commands/cmd_daemon.py

```diff
@@ -71,7 +71,7 @@ def start(foreground, number):
         subprocess.check_output(command, env=currenv, stderr=subprocess.STDOUT)  # pylint: disable=unexpected-keyword-arg
     except subprocess.CalledProcessError as exception:
         click.secho('FAILED', fg='red', bold=True)
-        echo.echo_critical(exception.output)
+        echo.echo_critical(str(exception))
```

Comment:
Or use `repr`, see https://stackoverflow.com/a/4308203/1069467

Reply:
I used `str` since that is what is intended for normal users, whereas `repr` is for debugging purposes. I think in this case, showing the exception type for the user is not necessary. But if you think this would actually be better, happy to change it

Reply:
I see... up to you.